controller.go

```diff
@@ -83,6 +94,10 @@ func (c *Controller) Process(m *messaging.Message) [][]byte {
 		}
 	}
 
+	if !c.HasRPCAccess(m.Source, req.Command) {
```

Comment:
Just a comment here. This should be turned into `HasAccess` in long term. Otherwise, we would have a lot of check here.